### ISSUE TITLE
Runs fail with Intel 16

### ISSUE THREAD

**Issue by reguly** [UK-MAC/BookLeaf_ref#1]
compiling with:
make MKFILEC=makefile.intel MOD=saltzmann bookleaf
(or sedov or sod), and with the appropriate input files, everything fails with Intel 16.0.0 20150815 + Intel MPI, with a cell volume < 0 error. Works fine with PGI.

**Comment by d-harris** [comment]
Hi Istvan,
which version are you using?
We did find an issue in getdt with the use of MINLOC:

The original code had a MINLOC over dtt, where dtt was gathered over the
procs and thus had an index from 0 to nproc-1.

However, with intel 16.0 it appears that minloc treats this array as
starting at 1, not zero, and so the wrong dt gets used. We fixed this in
the latest master and V1.2 tag by taking 1 off the minloc result.

See line 161 of getdt in there:

      ii=MINVAL(MINLOC(dtt))-1_ink  ! minloc converts (0:n) index to (1:n+1)


Hope that helps,
Duncan
